bin/cmd.js

```diff
@@ -254,7 +256,23 @@ else if (argv._[0] === 'set') {
             stamp = row.key.split('!')[1];
             prop = argv._[1];
             value = argv._.slice(2).join(' ');
-            set(stamp, prop, value);
+            terms = value.split(' ');
```

Comment:
I would try to not reinvent the wheel here: [parse-messy-time](https://github.com/substack/parse-messy-time) would fit perfectly.

```
> var parse = require('parse-messy-time')
> parse('now')
Mon Nov 16 2015 19:32:42 GMT+0100 (CET)
> parse('10 minutes ago')
Mon Nov 16 2015 19:22:42 GMT+0100 (CET)
> parse('10 min ago')
Mon Nov 16 2015 19:22:42 GMT+0100 (CET)
```